### PR TITLE
support npm 3 #104

### PR DESCRIPTION
**Pull request by hthetiot**
See https://github.com/montagejs/mr/pull/105#issuecomment-284619208 for details
cc @kriskowal

### CODE REVIEW

File: require.js [hunk 1]
```diff
@@ -724,7 +735,11 @@
 		}
         delete description.overlay;
 
-        config.packagesDirectory = URL.resolve(location, "node_modules/");
+        if (config.strategy === 'flat') {
+            config.packagesDirectory = URL.resolve(location, "/node_modules/");
```
Comment (kriskowal) [inline require.js]: root location.

**Comment by kriskowal** [comment]
would be good to capture a test.

**Comment by hthetiot** [comment]
@kriskowal here is your requested test
- 9a1f66b5213f39277073af67aba5031981dc35e7

**Comment by marchant** [comment]
hey @kriskowal, do you want to take a look to @hthetiot final change for that PR and merge if ready? Thanks!

**Pull request merged**